services/opentelemetry-operator/0.93.0/helmrelease/opentelemetry.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: OCIRepository
+metadata:
+  name: ghcr.io-opentelemetry-operator
+  namespace: ${workspaceNamespace}
+spec:
+  url: oci://ghcr.io/open-telemetry/opentelemetry-helm-charts/opentelemetry-operator
+  ref:
+    tag: 0.93.0
```

Comment:
One thing to consider is we might wanna put OTel Operator in kommander-apps repo because it [depends on prometheus operator](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/compatibility.md#compatibility-matrix)(KPS in k-apps). ofc we need to loop in product to finalize on this probably